include/aws/common/priority_queue.h

```diff
@@ -117,6 +117,12 @@ bool aws_priority_queue_backpointers_valid(const struct aws_priority_queue *cons
 AWS_COMMON_API
 bool aws_priority_queue_is_valid(const struct aws_priority_queue *const queue);
 
+/**
+ * Set of properties of a wiped aws_priority_queue.
+ */
+AWS_COMMON_API
+bool aws_priority_queue_is_wiped(const struct aws_priority_queue *const queue);
```

Comment:
Change to use the macros from https://github.com/awslabs/aws-c-common/pull/453

Reply:
Now using the AWS_IS_ZEROED macro